PluginsAndFeatures/azure-toolkit-for-intellij/src/com/microsoft/azure/hdinsight/spark/ui/SparkSubmissionContentPanel.kt

```diff
@@ -572,12 +579,17 @@ open class SparkSubmissionContentPanel(private val myProject: Project, val type:
             val uploadRootPath = storageWithUploadPathPanel.viewModel.getCurrentUploadFieldText()
                     ?.replace("/${Constants.submissionFolder}/?$".toRegex(), "")
 
+            // Currently we only support VFS for Gen2 storage account
+            if (uploadRootPath.isNullOrEmpty() || !AbfsUri.isType(uploadRootPath)) {
```

Comment:
Looks like we can change `isType`'s parameter to `@Nullable` to make this check simpler.

Reply:
Sure. Will fix for both WasbUri and AbfsUri

---
In reply to: [361566353](https://github.com/microsoft/azure-tools-for-java/pull/3885#discussion_r361566353) [](ancestors = 361566353)